src/main/java/com/app/rys/enums/BookingState.java

```diff
@@ -0,0 +1,13 @@
+package com.app.rys.enums;
+
+public enum BookingState {
+	ANULADA, PENDIENTE, CONFIRMADA;
+	
+	private String BookingState;
+	
```

Comment:
Te falta este constructor: 
BookingState(String bookingState) {
		this.bookingState=bookingState;
	}

Reply:
Hecho